Source/Krypton Components/TestForm/UserInputToastNotificationTest.cs

```diff
@@ -1,12 +1,274 @@
-﻿using Krypton.Toolkit;
+﻿using System;
```

Comment:
Why are these here ?

Reply:
Also These new files needs to have the License header added.

Reply:
Sorted